src/lifecycle/created.js

```diff
@@ -37,10 +37,15 @@ function notifyReady (elem) {
   });
 }
 
+function renderIfNotResolved (elem, opts) {
```

Comment:
Had to move the resolved checks to a function here. This used to be in the `renderer` lifecycle, but since we want to be able to re-use that and not have it check if it's resolved or not, we had to move this here.